src/stability/stabilityChecker.ts

```diff
@@ -10,7 +10,7 @@ const jobs = [
   },
   {
     method: collectorSync,
-    cron: '*/5 * * * * *'
+    cron: '0 * * * * *'
```

Comment:
By changing this, you will run collectorSync at 0:00 everyday.. Is this intended?

Reply:
it will run on 0 second on every minutes, before restarting collector it will will wait for 30 sec and check if block table is updating or not. if not then stability will restart the collector. that why will run the cron in every minutes.

Reply:
oh sorry I was misunderstood

Reply:
but why not * * * * *?

Reply:
But it's easy to read, no one has to bother if its 5 columns or 6 columns in the cron schedule. and also consistent with other usages in the repo.